docs/server-side/v2/how-to-guides/live-transcription-hls.mdx

```diff
@@ -0,0 +1,213 @@
+
+
+### Example Output
+
+The captions are generated in the `webvtt` format and are plugged in the HLS manifest itself. 
+
+#### Master manifest (master.m3u8)
+Following is the format of the master manifest file for HLS. 
+```
+#EXTM3U
+#EXT-X-VERSION:6
+#EXT-X-MEDIA:TYPE=SUBTITLES,GROUP-ID="subs",LANGUAGE="en-US",NAME="English CC (auto)",AUTOSELECT=YES,DEFAULT=YES,URI="subtitles/subtitles.m3u8",FORCED=NO,CHARACTERISTICS="public.accessibility.transcribes-spoken-dialog"
+#EXT-X-STREAM-INF:RESOLUTION=1280x720,BANDWIDTH=1720400,FRAME-RATE=20.000,SUBTITLES="subs"
+stream_0/stream.m3u8
+#EXT-X-STREAM-INF:RESOLUTION=854x480,BANDWIDTH=1005400,FRAME-RATE=20.000,SUBTITLES="subs"
+stream_1/stream.m3u8
+#EXT-X-STREAM-INF:RESOLUTION=640x360,BANDWIDTH=620400,FRAME-RATE=20.000,SUBTITLES="subs"
+stream_2/stream.m3u8
+```
+
+#### Subtitles manifest
+Following is the format of the manifest file for subtitles. 
+```
+#EXTM3U
+#EXT-X-VERSION:6
+#EXT-X-MEDIA-SEQUENCE:428
+#EXT-X-ALLOW-CACHE:NO
+#EXT-X-TARGETDURATION:2
+#EXTINF:2,
+output00001.vtt
+#EXTINF:2,
+output00002.vtt
+#EXTINF:2,
+output00003.vtt
+#EXTINF:2,
+output00004.vtt
+#EXTINF:2,
+output00005.vtt
+```
+
+#### `Webvtt` file format
+
+```
+WEBVTT
+X-TIMESTAMP-MAP=LOCAL:00:00:00.000,MPEGTS:0
+
+00:00:00.000 --> 00:00:01.022 position:50.00% size:80.00% align:middle
+If you if you're
+
+00:00:01.022 --> 00:00:02.000 position:50.00% size:80.00% align:middle
+If you if you are at some event here and you shoot the light ray,
+
+00:00:02.000 --> 00:00:03.065 position:50.00% size:80.00% align:middle
+If you if you are at some event here and you shoot the light ray,
+```
+
+</StepsContainer>
+
+
+### Frequently Asked Questions (FAQ)
+
+
+1. **Do live captions work within 100ms conferencing rooms (WebRTC based)?**
+    
+    Currently, live transcription is only supported within our HLS streams. 
+    
+2. **How many languages are supported?**
+    
+    Presently, only English language is supported. But support for other popular languages like French, Portuguese, Spanish and more is coming soon.
+    
+3. **Is live translation also supported?**
+    
+    Live translation is not supported right now.
+    
+4. **What happens if multiple languages are being spoken in the live stream?**
+    
+    The bits which are spoken in the selected language will be transcribed. There might be a few hallucinations though.
+    
+5. **When can I edit my live transcription configuration?**
+    
+    Live transcription configuration can be edited either on the room template, or at the start of a live stream.
+    
+6. **What all players are supported?**
+    
+    Almost all HLS players will support `webvtt` based transcripts. We have internally tested with hls.js, AV Player, Avo player and Exoplayer.
+    
+7. **Do you support speaker labels with the live transcription?**
+    
+    No, we don’t support speaker labels in live transcription. We do offer speaker labelled transcription
```

Comment:
how are we branding post call transcription? 
live transcription vs transcription is still ambiguous. we can say transcriptions for recordings or transcriptions for pre recorded file etc - these are used across other players also.

Reply:
Fair, I think I left that sentence unfinished. We are calling it post-call transcription only (with speaker labels. Have corrected it and linked it to the post call transcription doc.